deploy/role.yaml

```diff
@@ -57,5 +57,6 @@ rules:
   - lunarway.com
   resources:
   - '*'
+  - postgresqldatabases
```

Comment:
Is this added by the generator? I would think the `'*'` would match it anyways?

Reply:
Yes, I didn't touch it. Maybe it's a good idea to spec the exact resources the SA grants the controller rights to inspect?